src/main/java/com/github/manolo8/darkbot/gui/MapDrawer.java

```diff
@@ -80,7 +83,22 @@ public void mousePressed(MouseEvent e) {
                     main.setRunning(!main.isRunning());
                     repaint();
 
-                } else main.hero.drive.move(mapGraphics.toGameLocation(e));
+                } else {
+                    Locatable loc = mapGraphics.toGameLocation(e);
+                    if (e.getClickCount() == 2) {
```

Comment:
how about right click to open a pop-up dropdown that has an option "jump to X" map? that way there's no possible confusion about it